packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/LiveRealm.kt

```diff
@@ -61,7 +61,8 @@ internal abstract class LiveRealm(val owner: RealmImpl, configuration: InternalC
         get() {
             // Initialize a new snapshot that can be reused until cleared again from onRealmChanged
             if (_snapshot.value == null) {
-                val snapshot = realmReference.snapshot(owner)
+                println("Create snapshot from owner")
```

Comment:
We could log this messages using `realm.log`

Reply:
This was debug output. Removing it.